src/ng/urlUtils.js

```diff
@@ -72,13 +78,19 @@ function urlResolve(url) {
 
   urlParsingNode.setAttribute('href', href);
 
+  var hostname = urlParsingNode.hostname;
+
+  if (!ipv6InBrackets && /[\da-e:]+/.test(hostname)) {
```

Comment:
Why not `a-f` in the RegExp?
Also, you probably meant `/^[...]+$/` (to match the whole hostname, not just a part of it).

Reply:
Off-by-one error! I broke `https://google.com` parsing. 😱 Fixed.